package.json

```diff
@@ -34,60 +34,55 @@
   "devDependencies": {
     "body-parser": "^1.17.1",
     "broccoli-asset-rev": "^2.4.5",
-    "broccoli-merge-trees": "^2.0.0",
```

Comment:
I think this was a dependency that Ember CLI used to add? I didn't find any usage in ESA for this.